CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 2.0.1 (Unreleased)
 
+* **New Resource:** `cloudflare_waf_group` ([#476](https://github.com/terraform-providers/terraform-provider-cloudflare/issues/476))
```

Comment:
we usually add this directly on master once it's merged, you don't need to worry about that in this PR.

Reply:
I'll remove this then!